.travis.yml

```diff
@@ -27,6 +27,10 @@ matrix:
           os: linux
           dist: xenial
           env: TOXENV=py38
+        - python: "3.9-dev"
+          os: linux
+          dist: xenial
```

Comment:
we could maybe also try some newer ubuntu for this.

Reply:
added job for focal fossa too